glide.yaml

```diff
@@ -1,7 +1,7 @@
 package: github.com/giantswarm/operatorkit
 import:
 - package: github.com/cenkalti/backoff
-  version: v1.1.0
+  version: master
```

Comment:
I am fine with pinning master for our libs but third party is not under our control and a tag might be better here. We also do not use master for the k8s libs. Just saying. Up to you.

Reply:
I thought WithMaxTires is not avialable in v1.1.0 but it is. Pinning back again.